learning_loop_node/detector/rest/about.py

```diff
@@ -1,26 +1,46 @@
 
-from typing import TYPE_CHECKING
+import sys
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Optional
 
 from fastapi import APIRouter, Request
 
+from ...data_classes import ModelInformation
+
 if TYPE_CHECKING:
     from ..detector_node import DetectorNode
+KWONLY_SLOTS = {'kw_only': True, 'slots': True} if sys.version_info >= (3, 10) else {}
 
 router = APIRouter()
 
 
-@router.get("/about")
+@dataclass(**KWONLY_SLOTS)
+class AboutResponse:
+    operation_mode: str = field(metadata={"description": "The operation mode of the detector node."})
+    state: Optional[str] = field(metadata={"description": "The state of the detector node."})
+    model_info: Optional[ModelInformation] = field(
+        metadata={"description": "The model information of the detector node."})
+    target_model: Optional[str] = field(metadata={"description": "The target model of the detector node."})
+    version_control: str = field(metadata={"description": "The version control of the detector node."})
```

Comment:
Adding "The {variable name} of the detector model" does not add information

Reply:
Yes but the description is shown in clear text on redoc (which is what I wanted to achieve)